perllib/FixMyStreet/Cobrand/Brent.pm

```diff
@@ -1357,6 +1357,13 @@ sub request_referral {
     # return 1 if ($data->{contamination_reports} || 0) >= 3; # Will be present on missing only
     return 1 if ($data->{how_long_lived} || '') eq '3more'; # Will be present on new build only
     return 1 if $data->{ordered_previously};
+
+    if (
+        (($data->{'container-choice'} && $data->{'container-choice'} == $CONTAINER_IDS{rubbish_grey_bin} && $data->{request_reason} eq 'extra')
+        || $data->{'container-' . $CONTAINER_IDS{rubbish_grey_bin}} && $data->{request_reason} eq 'extra')
+        && ($data->{request_property_people} == 6 || $data->{request_property_nappies} eq 'Yes')
+    )
```

Comment:
Oh hang on, sorry, one more important thing than anything else I've noted - *all* grey bins should be referred at best. The logic is "if <=5 people and no children, refuse, otherwise refer". See https://github.com/mysociety/societyworks/issues/5120#issuecomment-3554093397

Reply:
So if we get here and it's a grey bin, this should always be returning 1. And if property_people is 1 and children is No then we should have gone to the refusal page before we've got here.

Reply:
Added a test to confirm that the report is fixed so won't be sent in 0e4782ee8c31ffba1eec25451021924a76a26f25